search-app/src/cmr/search/models/query.clj

```diff
@@ -156,6 +156,11 @@
   [_]
   [{:field :tag-key :order :asc}])
 
+(defmethod common-qm/default-sort-keys :variable
+  [_]
+  [{:field :variable-name :order :asc}
```

Comment:
I am not working on sort within this PR, but this only defines the sort-key, the real elasticsearch field is actually variable-name.lowercase which is defined in `cmr.search.data.query-to-elastic`